apps/swagger/styles/globals.css

```diff
@@ -14,3 +14,117 @@ a {
 * {
   box-sizing: border-box;
 }
+
+/* .scheme-container {
```

Comment:
Let's remove comments

Reply:
Done! Also removed the ugly scrollbar in cusotm css for now, will update/fix in another PR